metadata-transformer/src/test/java/no/unit/nva/metadata/service/MetadataServiceTest.java

```diff
@@ -87,19 +92,19 @@ public void getCreatePublicationParsesHtmlAndReturnsMetadata(String html, Create
         CreatePublicationRequest request = requestWithContributorName(name);
 
         return Stream.of(
-                generateMetadataHtml(Map.of(
-                        DC_CONTRIBUTOR, name),
-                        request),
-                generateMetadataHtml(Map.of(
-                        DC_CONTRIBUTOR_UPPER_CASE, name),
-                        request),
-                generateMetadataHtml(Map.of(
-                        DC_CREATOR, name),
-                        request),
-                generateMetadataHtml(Map.of(
-                        DC_CREATOR_UPPER_CASE, name),
-                        request)
-                );
+            generateTestWithMetadataHtml("dc.contributor map to contributor name in request",
```

Comment:
It seems that the intention of of these tests is to test whether non-capital and capital prefixes are accepted. The intention of each line can be understood only by comparing it to another line in the test. It would be easier if they were self contained. Example: LOWER_CASE_DC_PREFIX_CONTRIBUTOR_MAP, UPPER_CASE_DC_PREFIX_CONTRIBUTOR_MAP